docs/tutorials/vqe_with_estimator.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<p>This cell takes a lot time to run. It should only create a maximum of 80 estimator jobs right? How long did it take for you to run this cell? Now I am a bit confused with the runtime session concept, with this context call, are all 80 estimator program sent as a single containerized job or 80 individual jobs? I think currently it's the latter case. If that is the case, this example does not really help skipping the queue right? All 80 jobs will have be queue seperately. Am I right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
This is probably a question for @rathishcholarajan @jyu00

Reply:
Yes, that is a good question. I saw discussion (I can't find it now, maybe it was in slack) about the built in VQE also being slow, so I'm not sure what the source of the problem is.

Reply:
The first job in a session should be expected to wait in the queue and once the session starts all other jobs in that session should not get queued.